README.md

```diff
@@ -8,16 +8,35 @@ The template for the path to load these scripts is:
 https://cdn.jsdelivr.net/gh/user/repo@version/file
 ```
 
+By excluding the tag, jsDelivr will fetch the latest semantic version of the script.
+
 Ending the file name with `.min.js` will prompt jsDelivr to automatically create and serve a minified version of the
 script.
 
 Example:
 ```html
-<script src="https://cdn.jsdelivr.net/gh/heyartifact/webflow/analytics.min.js"></script>
+<script src="https://cdn.jsdelivr.net/gh/heyartifact/webflow/src/analytics.min.js"></script>
 ```
 
 Refer to the [jsDelivr features](https://www.jsdelivr.com/features#gh) for more information.
 
+## Testing
+
+Merging script changes to `main` may affect the live page. To test these scripts without making any changes to the live
+pages, you can edit the scripts in the Webflow page to target a specific branch (such as `develop`):
+
+```html
+<script src="https://cdn.jsdelivr.net/gh/heyartifact/webflow@develop/src/analytics.min.js"></script>
+```
+
+**Remember to changes these back before publishing the Webflow page to production!**
+
+Note that the files are cached and will not automatically fetch new versions with future commits (unless it sees that
+the version has changed). However, if a branch is cached and is stale for testing, you can also use the commit id to
```

Comment:
I need to test to be sure, but my understanding is that when the version updates on GitHub through a tagged release it will fetch the latest version instead of using the cache.

I can create my own repo for some testing (to avoid adding random commits/releases in this repo).